client/.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
will travis build recursively? If so then may go ahead and change it, but if not may just leave this be until we are building with travis.